Solon_Burleson/LeadsandClients/static/main.js

```diff
@@ -0,0 +1,33 @@
+window.onload = function () {
+
```

Comment:
result = []
for (var i=0; I<leadsarr.length; i++){
	result.push({y:leadsarr[i].name, label:leadsarr[i].leads})
}